src/components/ui/creating/index.tsx

```
@@ -27,6 +38,94 @@ export default function CreatingModal({ onClose }: CreatingModalProps) {
+                },
+                files: imageUrls,
+                hashTags: hashtags
+            });
+
+            if (result) {
+                alertStore.success('게시물이 성공적으로 생성되었습니다!');
+                onClose();
+            } else {
+                alertStore.error('게시물 생성에 실패했습니다.');
+            }
+        } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)